variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default = "JP"
+}
+
+variable "docker_url" {
+}
+
+variable "flask_app" {
+}
+
+variable "flask_env" {
+}
+
+variable "secret_key" {
+  sensitive = true
+}
+
+variable "website_diagnostics_enabled" {
+  type = bool
+}
+
+variable "website_service_storage_enabled" {
+  type = bool
+}
```

Comment:
Some of these feel like they could have defaults set, so that your pipeline etc. usually doesn't have to think about it

Reply:
Thanks Jack

I changed these to defaults and removed them as variables from my pipeline.

website_diagnostics_enabled = "true"
website_service_storage_enabled = "false"